drivers/scheduler/k8s/k8s.go

```diff
@@ -2039,6 +2038,12 @@ func (k *K8s) addSecurityAnnotation(spec interface{}, configMap *corev1.ConfigMa
 						obj.Parameters[CsiNodePublishSecretNamespace] = configMap.Data[secretNamespaceKey]
 						obj.Parameters[CsiControllerExpandSecretNamespace] = configMap.Data[secretNamespaceKey]
 					}
+					if app.IsCSI {
```

Comment:
you are doing app.IsCSI check twice here, can this block be before line 2030 as a single if block ?

Reply:
This is required since the annotations will be needed only for CSI app.